specification/ilm/move_to_step/types.ts

```diff
@@ -18,7 +18,8 @@
  */
 
 export class StepKey {
-  action: string
-  name: string
+  // action and name are optional in case they are used in next_step
+  action?: string
+  name?: string
   phase: string
 }
```

Comment:
Should we make a `NextStepKey` class? Just wondering, really.

Reply:
was wondering the same, that would be a breaking change though and it would have to wait

Reply:
Agreed! Can you please add it to https://github.com/elastic/elasticsearch-specification/issues/2973?

And feel free to merge this pull request

Reply:
Not 100% related, but juding from the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-move-to-step.html), it seems that both, `current_step` and `next_step` should be required.

We currently model them as optional:

```ts
/**
 * @rest_spec_name ilm.move_to_step
 * @availability stack since=6.6.0 stability=stable
 */
export interface Request extends RequestBase {
  path_parts: {
    index: IndexName
  }
  body: {
    current_step?: StepKey
    next_step?: StepKey
  }
}
```

Reply:
good catch! I'll remove the optional